src/core/BufferAttribute.js

```diff
@@ -23,7 +23,7 @@ class BufferAttribute {
 		this.array = array;
 		this.itemSize = itemSize;
 		this.count = array !== undefined ? array.length / itemSize : 0;
-		this.normalized = normalized === true;
+		this.normalized = normalized;
```

Comment:
Can you also apply the same change here?

https://github.com/mrdoob/three.js/blob/309b00afb6dcbc5e6c58e72f10eaa8d2e8888c83/src/core/InterleavedBufferAttribute.js#L19